Regift/Regift.swift

```diff
@@ -293,14 +293,11 @@ public struct Regift {
 
         // Create a dispatch group to force synchronous behavior on an asynchronous method.
         let gifGroup = Group()
-        var dispatchError: Bool = false
         gifGroup.enter()
 
         generator.generateCGImagesAsynchronously(forTimes: times, completionHandler: { (requestedTime, image, actualTime, result, error) in
             guard let imageRef = image , error == nil else {
                 print("An error occurred: \(error), image is \(image)")
-                dispatchError = true
-                gifGroup.leave()
```

Comment:
I think that `gifGroup.leave()`'s position is bad. In this case, `leave` is called if error occures. We should call `leave` every cases.